bindings/wasm/bindings.cpp

```diff
@@ -32,17 +32,14 @@ Manifold Intersection(const Manifold& a, const Manifold& b) { return a ^ b; }
 
 Manifold UnionN(const std::vector<Manifold>& manifolds) {
   return Manifold::BatchBoolean(manifolds, Manifold::OpType::Add);
-  ;
 }
 
 Manifold DifferenceN(const std::vector<Manifold>& manifolds) {
   return Manifold::BatchBoolean(manifolds, Manifold::OpType::Subtract);
-  ;
 }
 
 Manifold IntersectionN(const std::vector<Manifold>& manifolds) {
   return Manifold::BatchBoolean(manifolds, Manifold::OpType::Intersect);
-  ;
```

Comment:
Regarding glMatrix types, [this](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/22770#issuecomment-356738192) is an interesting thread on how they think about types.

I also found this more complete set of [types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d022534a71572148357b37c8e4b9b8ec4d8b5f37/types/gl-matrix/index.d.ts) - it's officially deprecated, but I think it's still pretty good, at least as a place to start.

Reply:
Yeah I think maybe we should just add the deprecated type definitions to the editor for now, the official typescript file is missing a lot of things and not very useful in our case. Hope that their API did not evolve too much.